scripts/profiles/create-profile.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+API="${API_ORIGIN:-http://localhost:4741}"
+URL_PATH="/profiles"
+curl "${API}${URL_PATH}" \
+  --include \
+  --request POST \
+  --header "Content-Type: application/json" \
+  --header "Authorization: Token token=$TOKEN" \
+  --data '{
+    "profile": {
+      "user_id": "Practice",
```

Comment:
remove sending `user_id` in requests. `user_id` is currently stored correctly based on current user